### PR TITLE
cleaner README and removing unneeded docker-compose files

### PR DESCRIPTION
**Pull request by matzew**
Some doc updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,92 +25,35 @@ The _AeroGear UnifiedPush Server_ is a server that allows sending push notificat
...
-And in a second terminal:
-
-    ./bin/standalone.sh -c standalone-full-ha.xml -Djboss.node.name=node2 -Djboss.messaging.cluster.password=somepassword -Djboss.socket.binding.port-offset=200 -Djava.net.preferIPv4Stack=true
+The easiest way to get started is running our `plain` Linux container:
+```
+mvn clean install -DskipTests && docker run -p 18081:8080 -it aerogear/ups:plain 
+```
 
-Note: on OS X, you need to enable multicast first:
+than go to `http://localhost:18081/` to use the UPS.
```
Comment (pb82) [inline README.md]: `than` -> `then`

File: docker-compose/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+docker run -p 18081:8080 -it aerogear/ups:plain 
+```
+
+### Keycloak UPS 
+
+During the build of the project the `aerogear/ups:kc` is build. The image is protected with Keycloak, and for that we have `docker-compose` file, that can be easily executed like:
+
+```
+cd servers/keycloak
+mvn docker:run -Pcompose
```
Comment (pb82) [inline docker-compose/README.md]: Is this still required? We can also just run `docker run -p 8080:8080 -it aerogear/ups:kc`

File: docker-compose/docker-compose-v2.1.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ services:
   unifiedpushserver:
     image: docker.io/aerogear/unifiedpush-wildfly:1.3.0
     volumes:
-       - ./helper:/ups-helper
+       - ./helper:/ups-helper:z
```
Comment (pb82) [inline docker-compose/docker-compose-v2.1.yaml]: what does the `z` at the end mean?

Comment (matzew) [reply docker-compose/docker-compose-v2.1.yaml]: SELinux, for safer mounts, on linux 
(this setup - as you now know, anyway works on Linux, unless you do some fancy networking in docker)

**Comment by matzew** [comment]
@pb82 thanks for the read, I've update the docker-compse doc

**Pull request merged**